src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/chapterPage/viewModel/ChapterPageViewModel.kt

```diff
@@ -0,0 +1,54 @@
+package org.wycliffeassociates.otter.jvm.app.ui.chapterPage.viewModel
+
+import io.reactivex.rxkotlin.subscribeBy
+import io.reactivex.subjects.PublishSubject
+import javafx.collections.FXCollections
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.ChapterPageModel
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.model.Verse
+import org.wycliffeassociates.otter.jvm.app.ui.chapterPage.view.Contexts
+import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
+import tornadofx.*
+
+class ChapterPageViewModel : ViewModel() {
+    val model = ChapterPageModel()
+    var activeChapter = PublishSubject.create<Int>()
+    var selectedTab = PublishSubject.create<Contexts>()
+    val chapters = FXCollections.observableArrayList<String>()!!
+    val verses = FXCollections.observableArrayList<Verse>()!!
```

Comment:
You might be able to get rid of the !! and warnings by explicitly marking the type like `var activeChapter: PublishSubject<Int> = ...` for these variables